packages/warehouses/src/warehouseClients/BigqueryWarehouseClient.ts

```diff
@@ -459,7 +464,15 @@ export class BigqueryWarehouseClient extends WarehouseBaseClient<CreateBigqueryC
             default:
                 break;
         }
-        // otherwise return a generic error
-        return new WarehouseQueryError(getErrorMessage(error));
+        console.error(
+            `Unknown bigquery warehouse error reason: ${JSON.stringify(
+                error,
+                null,
+                2,
+            )}`,
+        );
+        return new WarehouseQueryError(
+            `Bigquery warehouse error: ${error?.reason}`,
```

Comment:
I'm not returning the message here , in case that contains sensitive information.